src/uwtools/tests/drivers/test_cdeps.py

```diff
@@ -0,0 +1,146 @@
+# pylint: disable=missing-function-docstring,protected-access,redefined-outer-name
+"""
+CDEPS driver tests.
```

Comment:
These tests are similar to those for other drivers, and execute the various task methods and helper methods in the module.

Reply:
@maddenp-noaa That is great. I was not aware of having test for drivers. Thanks for adding it.